Sustainsys.Saml2.AspNetCore2/Sustainsys.Saml2.AspNetCore2.csproj

```diff
@@ -26,8 +23,7 @@
 
   <ItemGroup Condition="!('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '6.0')))">
     <PackageReference Include="Microsoft.AspNetCore.Authentication" Version="2.1.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.Cookies" Version="2.1.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.1" />
```

Comment:
<PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.1" /> didnt seem to be used for anything , so removed it